website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,287 @@
+The Rich Tooltip component associates a "toggle" element with a "tooltip" element that wraps some generic content (in that sense, both elements act as "containers"). "Soft" (hover/focus) or "click" event listeners can be assigned to the toggle, and when triggered they toggle the visibility of the tooltip, which appears as "popover" on top of the page content. When the tooltip is visible, it can be closed in various ways: toggling via the "soft" or "click" events, clicking outside of the popover, or via the `esc` key.
+
+Now, while at first glance the Rich Tooltip component may seem a simple component, beneath the surface it conceals a lot of complexity:
+
+- the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) to display the tooltip content as "popover" on top of the page content
+- it also uses a third-party library called [Floating UI](https://floating-ui.com/) to provide the anchoring of the "popover" to the "toggle" element, and the automatic positioning/collision detection functionality.
+- these functionalities are in turn abstracted in two underlying HDS utilities (the `hds-anchored-position` modifier and the [PopoverPrimitive component](/utilities/popover-primitive)), on top of which the Rich Tooltip component is built.
+
+The component also provides a lot of options to customize its behaviour. Below we have tried to describe the most common ones, but if you find a use case that is not supported by the existing Rich Tooltip implementation, please [speak with the Design System team](/about/support).
```

Comment:
😀 I'm not sure if we've discussed whether to use the American spelling or the British spelling of "behavior"? Not opinionated on it, but we should ensure that we're doing to the same thing in other places.

```suggestion
The component provides a lot of options to customize its behavior. Below we have tried to describe the most common examples, but if you find a use case that is not supported by the existing Rich Tooltip implementation, please [speak with the Design System team](/about/support).
```